index.bs

```diff
@@ -217,36 +221,17 @@ urlPrefix: https://fetch.spec.whatwg.org; spec: FETCH;
       between Priority Hints and request prioritization. However, it does describe ways in which implementations are encouraged to
       influence a request's overall fetch priority given a Priority Hint.</p>
 
-    <strong>HTTP Stream Priority</strong>
```

Comment:
Why remove the H3 language?

Reply:
The HTTP extensible priorities scheme isn't necessarily limited to HTTP/3 though that's likely where it will mostly be used. There was talk about leveraging it for HTTP/2 since it is header-based and whatever comes after HTTP/3 may also leverage it.

Reply:
Argh. Sorry about that, mis-understood the ask.  Looks like the rebasing of my 2 branches wasn't clean and it reverted one of the changes from the previous PR.  Should be fixed now.

Reply:
Apologies for not being clearer!